src/central-analytics.rst

```diff
@@ -0,0 +1,22 @@
+
+.. _central-analytics:
```

Comment:
central-usage-information

Reply:
should i rename the file too then?

Reply:
Yeah, please rename. And I'm thinking central-usage-reporting instead of central-usage-information. Sorry for the back and forth. Might be good to hold off until we settle the discussion on Slack.